eloquentjs/3_1_minimum.js

```diff
@@ -0,0 +1,21 @@
+/**Returns the minimum value between two numeric input values */
+function minimum(a,b = Infinity) {
+    if (isNaN(a) || isNaN(b)) {
+        throw "Minimum input cannot be NaN";
+        return;
+    }
+    if (a < b) {
```

Comment:
I'd use elses here to make it more clear that at least one of these is returning. You can also use <= to simplify the logic